src/planet_mcp/servers/search.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
+
+from fastmcp import FastMCP
+from planet import Planet
+from planet_mcp import models
+from planet_mcp.servers import descriptions
+
+# technically not using the sdk to make the tool but we can add it to that
+
+mcp = FastMCP("sdk")
+
+
+@mcp.tool(
+    name="data_search",
+    description=descriptions.overrides["data_search"],
+    tags={"data", "search"},
+)
+async def data_search(
+    item_types: list[str],
+    start_date: str,
```

Comment:
```suggestion
    start_date: str | None,
```

the type hint (used for validation) should be updated to allow None